src/main/kotlin/com/sys1yagi/kmockito/mockito_extensions.kt

```diff
@@ -66,12 +66,12 @@ private fun <T> Class<T>.defaultMock(): T {
     val mockSettingsImpl = MockSettingsImpl<T>()
     mockSettingsImpl.defaultAnswer(Answers.RETURNS_DEFAULTS)
     val mockCreationSettings = mockSettingsImpl.confirm(this)
-    return MockUtil().createMock(mockCreationSettings)
+    return MockUtil.createMock(mockCreationSettings)
 }
 
 // answer
 
-inline fun <reified T : Any> InvocationOnMock.getArgumentAt(index: Int): T = getArgumentAt(index, T::class.java)
+inline fun <reified T : Any> InvocationOnMock.getArgumentAt(index: Int): T = getArgument(index)
```

Comment:
[IMO] This is no useful method any more, but hold it to backward compatibility.
If it isn't need, I'll remove this method.